blackbox/docs/release_notes/2.0.0.txt

```diff
@@ -299,3 +302,28 @@ sets both the minimum and maximum memory to allocate, and should be set to
 whatever your previous ``CRATE_MAX_MEM`` was set to.
 
 .. _enterprise edition: https://crate.io/enterprise-edition/
+
+Cluster name in path data
+-------------------------
+
+In previous versions, the cluster name was a folder inside the ``path.data``
+directory. Now, the data is written directly in the ``path.data`` directory.
+If it still contains a folder equals the cluster name it is read at startup.
```

Comment:
I would rephrase this to something like 'If the directory still contains a folder whose name is the same as the cluster name, it is read at start up.'